x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/agentless/cis_integration_gcp.ts

```diff
@@ -23,16 +23,10 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     let cisIntegration: typeof pageObjects.cisAddIntegration;
     let cisIntegrationGcp: typeof pageObjects.cisAddIntegration.cisGcp;
     before(async () => {
-      await pageObjects.svlCommonPage.login();
-
```

Comment:
It looks like ` await pageObjects.svlCommonPage.loginAsAdmin();` is missing here, am I missing smth?

Reply:
You mean this file needs to login as admin to function?

Reply:
In that case, should it not fail?

Reply:
I guess that's because the first test in the config is `cis_integration_aws.ts` and it already has `pageObjects.svlCommonPage.loginAsAdmin()` . While this works as logout code was removed it makes the test suit depend on each other and on their order. The login logic is implicit now. Reading the docs https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/README.md#roles-based-testing it reads `in each test file top level describe suite should start with loginWithRole call in before hook` which I'm missing here

Reply:
https://github.com/elastic/kibana/pull/188128/commits/badde66df75dea281bfa8a182ee1158b664eb894

Reply:
@maxcold hello again, may I have one more review please?